app/src/pages/inside/logsPage/logsGrid/logsGrid.jsx

```diff
@@ -0,0 +1,253 @@
+    intl: intlShape.isRequired,
+    logItems: PropTypes.arrayOf(PropTypes.object),
+    loading: PropTypes.bool,
+    filter: PropTypes.string,
+    onFilterChange: PropTypes.func,
+    sortingColumn: PropTypes.string,
+    sortingDirection: PropTypes.string,
+    onChangeSorting: PropTypes.func,
+    consoleView: PropTypes.bool,
+    markdownMode: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    logItems: [],
+    loading: false,
+    filter: '',
+    onFilterChange: () => {},
+    sortingColumn: '',
+    sortingDirection: '',
+    onChangeSorting: () => {},
+    consoleView: false,
+    markdownMode: false,
+  };
+
+  getColumns = () =>
+    this.props.consoleView
+      ? [
+          {
+            id: 'attachment',
+            component: AttachmentColumn,
+            customProps: {
+              consoleView: true,
+            },
+          },
+          {
+            id: timeColumnId,
+            sortable: true,
+            customProps: {
+              consoleView: true,
+              markdownMode: this.props.markdownMode,
+            },
+            component: MessageColumn,
+          },
+          {
+            id: 'mobileTime',
+            component: TimeColumn,
+            customProps: {
+              mobile: true,
+            },
+          },
+          {
+            id: 'mobileAttachment',
+            component: AttachmentColumn,
+            customProps: {
+              mobile: true,
+            },
+          },
+        ]
+      : [
+          {
+            id: 'logMessage',
+            title: {
+              component: LogMessageSearch,
+              componentProps: {
+                filter: this.props.filter,
+                onFilterChange: this.props.onFilterChange,
+              },
+            },
+            maxHeight: 200,
+            component: MessageColumn,
+            customProps: {
+              markdownMode: this.props.markdownMode,
+            },
+          },
+          {
+            id: 'attachment',
+            component: AttachmentColumn,
+          },
+          {
+            id: timeColumnId,
+            title: {
+              full: this.props.intl.formatMessage(messages.timeColumnTitle),
+            },
+            sortable: true,
+            component: TimeColumn,
+          },
+          {
+            id: 'mobileAttachment',
+            title: {
+              component: () => <div className={cx('no-header')} />,
+            },
+            component: AttachmentColumn,
+            customProps: {
+              mobile: true,
+            },
+          },
+        ];
+
+  getLogRowClasses = (value) => {
+    const { consoleView } = this.props;
+
+    return {
+      log: true,
+      'error-row': !consoleView && (value.level === ERROR || value.level === FATAL),
+      [cx('row-console')]: consoleView,
+      [cx('error-row-console')]: consoleView && (value.level === ERROR || value.level === FATAL),
+    };
+  };
+
+  renderConsoleViewHeader = () => {
```

Comment:
Can console view header be created using grid columns?

Reply:
Fixed